### ISSUE TITLE
Use live server with .htacess file removing .html

### ISSUE THREAD

**Issue by thrivincode** [tapio/live-server#203]
I used this in .htacess file to hide the ".html" in the URL to improve UX:

RewriteEngine On
RewriteCond %{REQUEST_FILENAME} !-f
RewriteRule ^([^\.]+)$ $1.html [NC,L]

Now I cannot use live server to click on links anymore as it cannot open the files without .html.
How to solve this? Can I embed my .htaccess file code into live server?

**Comment by allejo** [comment]
I second support for serving without the `.html` extension just like http-server does

**Comment by zhuzhenzhen** [comment]
I have a same problem， have you resolved it ？ @thrivincode

**Comment by winuxue** [comment]
any new?

**Comment by edelstone** [comment]
Yeah, this is basically the only thing preventing me from using live-server instead of http-server. Would love to see it added.

**Comment by sanjaypojo** [comment]
Are there plans to support this? Difficult to use live-server if all links are broken! Love the library otherwise.
